examples/python-guide/advanced_example.py

```diff
@@ -187,12 +189,10 @@ def accuracy(preds, train_data):
                 feval=[binary_error, accuracy],
                 valid_sets=lgb_eval)
 
-print('Finished 50 - 60 rounds with self-defined objective function '
-      'and multiple self-defined eval metrics...')
+print('Finished 50 - 60 rounds with self-defined objective function and multiple self-defined eval metrics...')
 
 print('Starting a new training job...')
 
-
```

Comment:
Please revert this whitespace change by adding back a blank line.

Reply:
Oh I see, that's why it was not passing one check. 
Thank you for mentioning I will surely do this.